modules/postgres/testdata/docker-entrypoint-ssl.bash

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
IIUC this shell script is to force the ownership in the cert files, right? What will happen if users don't do it to their projects?

Reply:
For postgres images, the container will fail to launch. 

I tried various file modes to get postgres to launch when the files are owned by root, but none of them worked. This is contra the docs where it appears root can own the docs.

The original reporter of this issue tried various approaches as well - https://github.com/testcontainers/testcontainers-go/issues/2404

Reply:
We could do that. If the user is enabling ssl then inject the entrypoint for them.

Reply:
I'm not sure if we want to expose a `WithEntrypoint`, at least for the modules, as it would allow changing the behaviour of a given module. I'd like to know when/if this is needed 🤔 Else, the module could behave differently from the expected behaviour, and can cause issues to users. For advanced uses cases, they can override the entire container request (see https://golang.testcontainers.org/modules/postgres/#customising-the-containerrequest)

Reply:
Okay, I wont export it.